import-service/src/handlers/catalogBatchProcesss.js

```diff
@@ -0,0 +1,18 @@
+const AWS = require('aws-sdk');
+
+exports.catalogBatchProcesss = (event, context) => {
+    const data = event.Records.map(record => record);
```

Comment:
**TASK 6.2**: "The lambda function should iterate over all SQS messages and create corresponding products in the products table."
I think you've missed this point. Evaluation criteria doesn't include this requirement but I think its the important part of this task and worth to be implemented 🙂